jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/steps/AttachDebugger.kt

```diff
@@ -73,6 +98,7 @@ class AttachDebugger(
     }
 
     private companion object {
+        const val endText = "END RequestId: "
```

Comment:
do we have a place noting all of the magic SAM strings we're relying on?

Reply:
Currently they are where they are used, but maybe we should move them into SamExecutable, thoughts?

Reply:
yeah i like that idea